docs/reference/all_strategies.rst

```diff
@@ -135,6 +135,9 @@ Here are the docstrings of all the strategies in the library.
 .. automodule:: axelrod.strategies.shortmem
    :members:
    :undoc-members:
+.. automodule:: axelrod.strategies.selfsteem
+  :members:
```

Comment:
Your indentation levels aren't correct here. Compare them to the other strategies.